osbuild/util/term.py

```diff
@@ -0,0 +1,32 @@
+"""Wrapper module for output formatting."""
+
+import sys
+
+from typing import Dict
+
+
+class VT:
+    """Video terminal output, disables formatting when stdout is not a tty."""
+
+    isatty: bool
+
+    escape_sequences: Dict[str, str] = {
+        "reset": "\033[0m",
+
+        "bold": "\033[1m",
+
+        "red": "\033[31m",
+        "green": "\033[32m",
+    }
+
+    def __init__(self) -> None:
+        self.isatty = sys.stdout.isatty()
```

Comment:
I think I would have done the `isatty` addition in a follow-up, so we first get the extraction into the module. But I don't mind either way. However, note that you now hard-code `stdout` in the module, while any code *using* this module might not necessarily know that it is specific to `stdout`. Just something to keep in mind. Not sure how to avoid this. One option would be to name the global instance to `stdout`/etc., but not sure about it.

Reply:
I thought I mentioned this somewhere but yea it's hard to figure out. Perhaps removing the isatty check is 'just better'? :)